docs/sources/configuration/_index.md

```diff
@@ -2384,7 +2384,7 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 # CLI flag: -frontend.min-sharding-lookback
 [min_sharding_lookback: <duration> | default = 0s]
 
-# Split queries by an interval and execute in parallel, any value less than zero disables it.
+# Split queries by a time interval and execute in parallel, a value of 0 disables splitting by time.
```

Comment:
Thanks for the clarification.  My suggestion improves the grammar of this PR.

```suggestion
# Split queries by a time interval and execute in parallel. The value 0 disables splitting by time.
```

Reply:
Lgtm, thanks!